### PR TITLE
Added support for multiple SDKs.

### PR DESCRIPTION
**Pull request by meletev**

### CODE REVIEW

File: src/main/java/eu/europa/ted/eforms/sdk/annotation/SdkComponent.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import eu.europa.ted.eforms.sdk.component.SdkComponentTypeEnum;
+
+@IndexAnnotated
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface SdkComponent {
+  public static final String ANY = "any";
+
+  public String version() default ANY;
```
Comment (rousso) [inline src/main/java/eu/europa/ted/eforms/sdk/annotation/SdkComponent.java:18]: Whats does this mean in practice? ANY SDK version?! 
It does not seem to make any sense.
Maybe if you would say LATEST there would be an argument in favor of it, but still I would not see a scenario where not specifying an SDK version explicitly makes any sense.

Maybe the solution here is to set it to UNDEFINED by default and then throw an exception if it has not been explicitly set to a specific version.

File: pom.xml [hunk 2]
```diff
@@ -42,6 +42,8 @@
     <efx.toolkit.version>0.1.2-SNAPSHOT</efx.toolkit.version>
 
     <!-- Third-party libraries -->
+    <antlr4.version>4.9.3</antlr4.version>
```
Comment (rousso) [inline pom.xml:45]: Why do we need to add these dependencies as explicit?

**Review by rousso — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (meletev) [reply pom.xml:45]: They were already explicit.
I just moved them to the "<properties>" section as a good practice when it comes to maintaining a POM.

File: src/main/java/eu/europa/ted/eforms/sdk/annotation/SdkComponent.java (hunk 1 above)
Comment (meletev) [reply src/main/java/eu/europa/ted/eforms/sdk/annotation/SdkComponent.java:18]: The "ANY" version is for those implementation classes that are not explicitly tied to a certain SDK version and can work with all of them. We might not have such classes right now, but they can occur.
Also, "ANY" currently serves as a fallback for when an implementation class is not found for the target SDK version. In this sense, if required, we can have classes with "this method is unsupported" code or some generic behaviour.

**Pull request merged**